_posts/2017-06-15-grpc-load-balancing.md

```diff
@@ -0,0 +1,190 @@
+## Introduction
+
+This document describes various load balancing scenarios seen when deploying gRPC. If you use [gRPC](http://grpc.io) with multiple backends, this document is for you.
+
+A large scale gRPC deployment typically has a number of identical back-end instances, and a number of clients. Each server has a certain capacity. Load balancing is used for distributing the load from clients optimally across available servers.
+
+### Why gRPC?
```

Comment:
Oh, sorry, the "large scale gRPC deployment" bit makes perfect sense, it was more the ### Why gRPC? bit. It's a very general "advantages of gRPC" section and I think it'd fit better if it actually mentioned load balancing, even in passing.